### PR TITLE
Add h.json

### PR DESCRIPTION
**Pull request by thaihwng**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/h.json [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+        "email": "",
+        "discord": "913402397588611162",
+        "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.DlgVUxlDvBwmitiShjGH7v07E-3PzBajhoPLtMY8VqUFaK4Dlk-2Vtbf-prPmKLj26MR7TuyiPP6vG5HpBNhJmymmFs0p5h_PICI-OJqAtX4dJq1sESiK9Rpbt88vXkiXDMD0vUJWJajdGpGz__dwRawy6iINyMQVVtuKrD41m_VfV6uYn8I4OekB3M0C9Y3eEOVEd113MFO4uPdzYYqjT_6GWI0kE0ra9mOiN10qRytituc9TANADx_6STVZHl_6sMF7v11Ui2YcJctAlpajknKKpg1T9dY6OuR7KyTzFpbABsE3x-oHfQa8hGte-5zk8F2ZFlWgGNkj081uVJkmg.eQeHQqoJp2uT8vHXcOLtnA.Nh511lf5K9YkvaGfQKtKucwSpyp3Kb98w0vCylRd5u2gvW3QcUVxGwLyHmLnCHEAAqrSVvtj2PZAUPN06Fhx9CTzIKmHIBS7B3XMQodjiDM.hL7u4luGeaqfQfZI83HXGw"
+    },
+    "record": {
+        "CNAME": "thaihwng.github.io",
+        "NS": [
+            "dilbert.ns.cloudflare.com",
+            "fish.ns.cloudflare.com"
+        ]
```
Comment (CuteDog5695) [inline domains/h.json:13]: You need to remove CNAME record since you want to use NS records.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
Also, since you want to use NS records as far as I see, you need to give a reason why you want it.

**Comment by thaihwng** [comment]
> Also, since you want to use NS records as far as I see, you need to give a reason why you want it.

I am switching to Cloudflare's NS records to enable custom domain configurations.

**Comment by thaihwng** [comment]
I deleted it by mistake!!

**Pull request closed without merging**